staging/src/k8s.io/api/events/v1/types.go

```diff
@@ -39,28 +40,28 @@ type Event struct {
 
 	// reportingController is the name of the controller that emitted this Event, e.g. `kubernetes.io/kubelet`.
 	// This field cannot be empty for new Events.
-	// +optional
+	// required as per the strict validation checks being performed over Event resource corrersponding to events.k8s.io/v1 or newer.
```

Comment:
It's actually the other way around - it's not required because we added validation - we added validation because it is required (the comments were just incorrect).

Reply:
ohh my bad .. will update them

Reply:
I believe I would just add "Required". That should be enough, i suppose